docs/reference/announcements.md

```diff
@@ -132,6 +132,27 @@ Starting with Camunda 8.6, the Camunda Optimize artifact has been split into two
 
 Make sure to update your Docker configurations accordingly to ensure compatibility.
 
+### New base path for Operate and Tasklist Webapps
+
+We are introducing a new base path for both the Operate and Tasklist **web applications**. This change applies to both self-managed and SaaS environments.
```

Comment:
Do we mean the Operate and Tasklist **components**?

Reply:
Hi @christinaausley,
Using “components” might sound too broad. I want to emphasis that this is applicable to the web interfaces urls (Operate and Tasklist UIs) and not to the Operate and Tasklist API urls, so I think "web applications" word is more appropriate.

Reply:
@christinaausley I think we should use "web applications" instead of "components". WDYT?

Reply:
@akeller I feel like we've had a handful of discussion on this, what do you think about this particular instance? CC @camunda/tech-writers

Reply:
Good spot @christinaausley, but I really like @houssain-barouni's reasoning: 

> I want to emphasis that this is applicable to the web interfaces urls (Operate and Tasklist UIs) and not to the Operate and Tasklist API urls

A component consists of a UI and an API, but a web app makes sense only when we talk about the UI. **Component** remains the product/brand name, but at the technical level, it's a **web application**. Great discussion!

Reply:
Thanks @akeller , I will replace "component" by "web application"

Reply:
Internal alignment happening [here](https://camunda.slack.com/archives/C0484UFR2VA/p1727441682445649).

Reply:
@christinaausley if we can't get alignment next week, let's not consider this blocking. I do want clear definitions, but it shouldn't block merging this.